README.md

```diff
@@ -15,6 +16,13 @@ Create a virtualenv (but you'd better use virtualenvwrapper or pew):
 
     virtualenv banenv --python=`which python3.4`
     source banenv/bin/activate
+    
+Install developpement tools
+    
+    pip install ipython
+    pip install ipdb
+    pip install setuptools
```

Comment:
ipython and ipdb still are useful and can be suggested, but setuptools install should not be needed, because it's already handled by the virtualenv. :)